about/troubleshooting.rst

```diff
@@ -0,0 +1,84 @@
+
+This is a `known bug <https://github.com/godotengine/godot/issues/20566>`__ on
+Windows when having specific USB peripherals connected. This issue has been
+reproduced in particular with Corsair's iCUE software. Try updating your USB
+peripherals' drivers to their latest version. If all else fails, disconnect them
+before starting the project then connect them again.
+
+The Godot editor appears to have frozen after clicking the system console.
+--------------------------------------------------------------------------
+
+When running Godot on Windows with the system console enabled, if you click
+inside the command window, you may have accidentally enabled *selection mode*.
+This Windows-specific behavior will pause the application to let you select text
+inside the system console. The application cannot override this behavior.
+
+To solve this, select the system console window and press Enter to leave
+selection mode.
+
+The project window appears blurry (unlike the editor).
+------------------------------------------------------
+
+Unlike the editor, the project isn't marked as DPI-aware by default. This is
+done to improve performance, especially on integrated graphics where rendering
+3D scenes in hiDPI is slow.
+
+To resolve this, open **Project > Project Settings** and enable
+**Display > Window > Dpi > Allow Hidpi**. On top of that, make sure your project
+is configured to support :ref:`multiple resolutions <doc_multiple_resolutions>`.
+
+The project window doesn't appear centered when I run the project.
+------------------------------------------------------------------
+
+This is a `known bug <https://github.com/godotengine/godot/issues/13017>`__.
+To resolve this, open **Project > Project Settings** and enable
+**Display > Window > Dpi > Allow Hidpi**. On top of that, make sure your project
+is configured to support :ref:`multiple resolutions <doc_multiple_resolutions>`.
+
+The project works when run from the editor, but fails to load some files when running from an exported copy.
+------------------------------------------------------------------------------------------------------------
+
+This is usually caused by forgetting to specify a filter for non-resource files
+in the Export dialog. By default, Godot will only include actual *resources*
+into the PCK file. Some files commonly used, such as JSON files, are not
+considered resources. For example, if you load ``test.json`` in the exported
+project, you need to specify ``*.json`` in the non-resource export filter.
+See :ref:`doc_exporting_projects_export_mode` for more information.
+
+On Windows, this can also be due to
+:ref:`case sensitivity <doc_project_organization_case_sensitivity>` issues.
+If you reference a resource in your script with different casing than on the
+filesystem, it will fail once the project is exported. This is because the
+virtual PCK filesystem is case-sensitive, unlike the Windows filesystem which is
+case-insensitive by default.
```

Comment:
```suggestion
The project works when run from the editor, but fails to load some files when running from an exported copy.
------------------------------------------------------------------------------------------------------------

This is usually caused by forgetting to specify a filter for non-resource files
in the Export dialog. By default, Godot will only include actual *resources*
into the PCK file. Some files commonly used, such as JSON files, are not
considered resources. For example, if you load ``test.json`` in the exported
project, you need to specify ``*.json`` in the non-resource export filter. See
:ref:`doc_exporting_projects_export_mode` for more information.

On Windows, this can also be due to :ref:`case sensitivity
<doc_project_organization_case_sensitivity>` issues. If you reference a resource
in your script with a different case than on the filesystem, loading will fail
once you export the project. This is because the virtual PCK filesystem is
case-sensitive, while Windows's filesystem is case-insensitive by default.
```